RecoTauTag/RecoTau/plugins/PFRecoTauDiscriminationByIsolation.cc

```diff
@@ -195,10 +228,29 @@ class PFRecoTauDiscriminationByIsolation : public PFTauDiscriminationProducerBas
   double offsetRelativeSumPt_;
   double customIsoCone_;
   
+  bool applyPhotonPtSumOutsideSignalConeCut_;
+  double maxAbsPhotonSumPt_outsideSignalCone_;
+  double maxRelPhotonSumPt_outsideSignalCone_;
+
+  bool applyFootprintCorrection_;
+  struct footprintCorrectionType
+  {
+    footprintCorrectionType(const std::string& selection, const std::string& offset)
+      : selection_(selection),
+	offset_(offset)
+    {}
+    ~footprintCorrectionType() {}
+    StringCutObjectSelector<PFTau> selection_;
+    StringObjectFunction<PFTau> offset_;
+  };
+  std::vector<footprintCorrectionType*> footprintCorrections_;
```

Comment:
you don't really need to put anything in the initializer list if the constructor body has the value intialized in any case